datajunction-query/.env

```diff
@@ -0,0 +1,3 @@
+CONFIGURATION_FILE="/code/config.djqs.yml"
+DEFAULT_REFLECTION_ENGINE=trino
```

Comment:
Can this be simplified to `DEFAULT_ENGINE`?

Reply:
This attribute already existed from a previous PR, I just set it here for the demo environment. It's used in a couple of places so I'll follow up with a separate PR to change this everywhere.

Reply:
Ah ok, it's fair to just leave it then!